sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorInstaller.java

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.opentelemetry.exporter;
+
+import com.azure.monitor.opentelemetry.exporter.implementation.AzureMonitorExporterProviderKeys;
+import com.azure.monitor.opentelemetry.exporter.implementation.AzureMonitorLogRecordExporterProvider;
+import com.azure.monitor.opentelemetry.exporter.implementation.AzureMonitorMetricExporterProvider;
+import com.azure.monitor.opentelemetry.exporter.implementation.AzureMonitorSpanExporterProvider;
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdkBuilder;
+import io.opentelemetry.sdk.metrics.Aggregation;
+import io.opentelemetry.sdk.metrics.InstrumentSelector;
+import io.opentelemetry.sdk.metrics.View;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+/**
+ * Use @link AutoConfiguredOpenTelemetrySdkBuilder} to install the azure monitor exporters.
+ */
+public final class AzureMonitorInstaller {
+
+    private final AutoConfiguredOpenTelemetrySdkBuilder autoConfiguredOpenTelemetrySdkBuilder;
+    private final AzureMonitorExporterBuilder azureMonitorExporterBuilder = new AzureMonitorExporterBuilder();
+
+    /**
+     * Construct an instance of AzureMonitorInstaller
+     * @param autoConfiguredOpenTelemetrySdkBuilder {@link AutoConfiguredOpenTelemetrySdkBuilder} installs the azure monitor exporters.
+     */
+    public AzureMonitorInstaller(AutoConfiguredOpenTelemetrySdkBuilder autoConfiguredOpenTelemetrySdkBuilder) {
+        this.autoConfiguredOpenTelemetrySdkBuilder = autoConfiguredOpenTelemetrySdkBuilder;
+    }
+
+    /**
+     * Sets the connection string to use for exporting telemetry events to Azure Monitor.
+     *
+     * @param connectionString The connection string for the Azure Monitor resource.
+     * @return the updated {@link AzureMonitorInstaller} object.
+     */
+    public AzureMonitorInstaller connectionString(String connectionString) {
+        azureMonitorExporterBuilder.connectionString(connectionString);
+        return this;
+    }
+
+    /**
+     * Customize the {@link AzureMonitorExporterBuilder} which is used to during {@link #install()}.
+     * @param exporterCustomizer the exporterCustomizer value to set.
+     * @return the updated {@link AzureMonitorInstaller} object.
+     */
+    public AzureMonitorInstaller exporterCustomizer(Consumer<AzureMonitorExporterBuilder> exporterCustomizer) {
+        exporterCustomizer.accept(azureMonitorExporterBuilder);
+        return this;
+    }
+
+    /**
+     * Configures an {@link AutoConfiguredOpenTelemetrySdkBuilder} based on the options set in the builder.
+     */
+    public void install() {
```

Comment:
I'm torn between passing `AutoConfiguredOpenTelemetrySdkBuilder` and passing it here. I think I prefer passing it here to make `install()` look less like some global state affecting method (no params, no return value).

thoughts? @heyams @jeanbisutti @lmolkova 

```suggestion
    public void install(AutoConfiguredOpenTelemetrySdkBuilder autoConfiguredOpenTelemetrySdkBuilder) {
```

Reply:
a static install or bring back AzureMonitorExporter.builder()? @lmolkova thoughts please.

Reply:
ya, that's the benefit of passing it in the constructor, because then we can have convenience method

```
static void install(AutoConfiguredOpenTelemetrySdkBuilder autoConfiguredOpenTelemetrySdkBuilder)
```

we could alternatively introduce another class, e.g.

```
AzureMonitor.installer()
    .connectionString(..)
    .install();

AzureMonitor.install(AutoConfiguredOpenTelemetrySdkBuilder);
```

and make AzureMonitorInstaller constructor package-private

Reply:
let me try it.